azkaban-webserver/src/test/java/azkaban/fixture/VelocityContextTestUtil.java

```diff
@@ -0,0 +1,26 @@
+package azkaban.fixture;
+
+import azkaban.utils.WebUtils;
+import org.apache.velocity.VelocityContext;
+import org.apache.velocity.tools.generic.EscapeTool;
+
+
+/**
+ * The type Velocity context test util.
+ */
+public class VelocityContextTestUtil {
+  /**
+   * Gets an instance of the velocity context
+   *
+   * Add the utility instances that are commonly used in many templates.
+   *
+   * @return the instance
+   */
+  public static VelocityContext getInstance() {
+    VelocityContext context = new VelocityContext();
+    context.put("esc", new EscapeTool());
+    WebUtils utils = new WebUtils();
+    context.put("utils", utils);
```

Comment:
No, it is not necessary for this change.
However this is here to mimic the production code in anticipation of future tests of .vm files that need this object.